ee/codegen/src/project.ts

```diff
@@ -779,4 +779,15 @@ ${errors.slice(0, 3).map((err) => {
       sandboxInputs: this.sandboxInputs ?? [],
     });
   }
+
+  /**
+   * Gets the generic node file paths that have been tracked during code generation
+   * @returns Object mapping file paths to booleans
+   */
+  public getGenericNodeFiles(): Record<string, boolean> {
```

Comment:
[nit] would love to update an existing test we have in `project.test.ts` that has generic nodes to assert that this is now returning a non-empty. and also one _without_ generic node's to show that it is empty